MAUI/Calendar/customizations.md

```diff
@@ -0,0 +1,257 @@
+        this.Calendar.MonthView.DisabledDatesTextStyle = textStyle;
+        this.Calendar.MonthView.SpecialDatesBackground = Colors.Purple;
+        this.Calendar.MonthView.SpecialDatesTextStyle = textStyle;
+        this.Calendar.MonthView.WeekendDatesBackground = Colors.Red;
+        this.Calendar.MonthView.WeekendDatesTextStyle = textStyle;
+
+{% endhighlight %}
+{% endtabs %}
+
+![monthview-customization-in-maui-calendar](images/customization/monthview-customization-in-maui-calendar.png)
+
+>**NOTE**
+* The Background color and text style will be applied based on the following order: selectableDayPredicate dates, special dates, disable dates, today date, trailingLeading dates, and weekend dates.
+
+## Month View Header format
+You can customize the month format of the `Calendar` by using the `TextFormat` property.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                      View="Month">
+            <Calendar:SfCalendar.MonthView>
+                <Calendar:CalendarMonthView>
+                    <Calendar:CalendarMonthView.HeaderView>
+                        <Calendar:CalendarMonthHeaderView TextFormat="ddd" />
+                    </Calendar:CalendarMonthView.HeaderView>
+                </Calendar:CalendarMonthView>
+            </Calendar:SfCalendar.MonthView>
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.MonthView.HeaderView = new CalendarMonthHeaderView()
+{
+    TextFormat = "ddd",
+};
+
+{% endhighlight %}
+{% endtabs %}
+
+![view-header-text-format-in-maui-calendar](images/customization/view-header-text-format-in-maui-calendar.png)
+
+## Year cell customization
+You can customize the calendar `year`, `decade`, and `century` view by using the `YearView` property of `SfCalendar`.
+
+*   **Year cell dates** – You can customize the year view date's text style and background of the `Calendar` by using the `TextStyle` and `Background` properties of `YearView`.
+
+*    **Disabled dates** – You can disable the date by using the MinimumDate, MaximumDate, EnablePastDates and SelectableDayPredicate callback. The date before `MinimumDate` is said to disabled date, the date after the `MaximumDate` is said to disabled date, if you want disable the date before today date by using the `EnablePastDates` and if you want to disable any particular date by using `SelectableDayPredicate` property. You can also customize the disabled dates text style and background of the `Calendar` by using the `DisabledDatesTextStyle` and `DisabledDatesBackground` properties of `YearView`.
+
+*    **Leading dates** – You can hide the Leading date by using the `ShowTrailingAndLeadingDates` property in the `SfCalendar` class. You can also customize the Leading dates text style and background of the `Calendar` by using the `LeadingDatesTextStyle` and `LeadingDatesBackground` properties of `YearView`. It is not applicable for Year view.
```

Comment:
You can hide the leading dates by using the `ShowTrailingAndLeadingDates` property in the `SfCalendar` class. You can also customize the leading dates text style and background of the `Calendar` by using the `LeadingDatesTextStyle` and `LeadingDatesBackground` properties of `YearView.`

Reply:
Content changed